scripts/sync-up-helm-chart.sh

```diff
@@ -32,7 +35,36 @@ if [ -n "${crd_diff}" ]; then
   cp -f $INSTALL_CRDS_YAML $CHART_FEDERATED_CRD_DIR/crds.yaml
 fi
 
+# Generate kubeconfig to access kube-apiserver. It is cleaned when script is done.
+cat <<EOF > ${WORKDIR}/kubeconfig
+apiVersion: v1
+clusters:
+- cluster:
+    server: 127.0.0.1:8080
+  name: development
+contexts:
+- context:
+    cluster: development
+    user: ""
+  name: federation
+current-context: ""
+kind: Config
+preferences: {}
+users: []
+EOF
+
+# Start kube-apiserver to generate CRDs
+./bin/etcd &
```

Comment:
I believe etcd will put its files in the federation path if invoked in this way.  Please ensure that it is started in `WORKDIR` instead.

```bash
pushd "${WORKDIR}"
  ${ROOTDIR}/bin/etcd &
popd
```

Reply:
Yes. there has a data directory in CWD. It is not expected. Fixed:
```diff
-./bin/etcd &
+./bin/etcd --data-dir ${WORKDIR} &
```